README.md

```diff
@@ -64,12 +70,11 @@ Ouput example:
 - Document your API if you are willing to.
 
 
-### Deliverable
+## Deliverable
 
 
-- A git repo hosted wherever you like.
+- It would be REALLY nice if it was hosted in a git repo of your own. You can create a new empty project, create a branch and Merge Request it to the master branch. Provide the MR URL for us. BUT if you'd rather, just compress this directory and send it back to us.
```

Comment:
Maybe better to use "Pull request" and "PR", since the code is on github.

Reply:
Yes, github use PULL REQUEST

Reply:
good one